src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,11 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery one = new Lottery();
```

Comment:
You can generate as many balls as you like with one Lottery object, why do you need three?